### PR TITLE
Fix network tab UI for smaller screens (bsc#968694)

### PR DESCRIPTION
**Pull request by jsuchome**
Correctly set initial state of VLAN ID field
- 3.1.18

### CODE REVIEW

File: src/include/crowbar/complex.rb [hunk 1]
```diff
@@ -47,6 +47,9 @@ def initialize_crowbar_complex(include_target)
 
       Yast.include include_target, "crowbar/helps.rb"
 
+      display_info = UI.GetDisplayInfo
```
Comment (shundhammer) [inline src/include/crowbar/complex.rb]: NP: This could be shortened to

  size_y = UI.GetDisplayInfo["Height"].to_i

**Comment by shundhammer** [comment]
Other than that nitpicking issue, LGTM.

File: src/include/crowbar/complex.rb (hunk 1 above)
Comment (jsuchome) [inline src/include/crowbar/complex.rb]: ok, let's do that

File: src/include/crowbar/complex.rb [hunk 2]
```diff
@@ -1567,14 +1571,15 @@ def get_tabs_descr
             HSpacing(2),
             VBox(
               "network_help",
+              @small_screen ? Empty() : VSpacing(),
```
Comment (mvidner) [inline src/include/crowbar/complex.rb]: Are you sure you want to keep the *other* VSpacing under this line?

Comment (jsuchome) [inline src/include/crowbar/complex.rb]: Good catch

**Comment by mvidner** [comment]
LGTM

**Pull request merged**